dependencies/pom.xml

```diff
@@ -123,7 +123,8 @@
         <version.lib.narayana>5.12.0.Final</version.lib.narayana>
         <version.lib.netty>4.1.73.Final</version.lib.netty>
         <version.lib.netty-io_uring>0.0.8.Final</version.lib.netty-io_uring>
-        <version.lib.oci-java-sdk-objectstorage>2.3.0</version.lib.oci-java-sdk-objectstorage>
+        <version.lib.oci>2.19.0</version.lib.oci>
```

Comment:
We should document the procedure for how our customers/developers can bump this to the latest oci version; which for now presumably is to let our team know through slack or something and then we publish a new release?  If that is indeed the case we should look at providing a means to have this decoupled from us, so we are agnostic to the version of the OCI SDK our customers pick in the future - they publish weekly.

Reply:
As discussed offline, Maven dependency management can be used for this